INSTRUCTIONS.native.md

```diff
@@ -0,0 +1,93 @@
+The following instructions are for setting up MSVC without docker.
+
+# FAQ
+
+## Does it run on Ubuntu 18.04 LTS?
+
+Yes, but the install scripts won't work because `msitools` is too old. You'll need to install either via Docker or on a real Ubuntu 20.04 LTS machine; and later copy paste the files under `/opt/msvc`
+
+## Does it work with CMake?
+
+Yes, but a custom version is needed or else CMake will complain that `/opt/msvc/bin/x64/cl` can't build a simple program.
+
+## Can it build Debug versions?
+
+Yes, but there may be troubles. From wine errors appearing in the logs/console to problems when trying to launch it because of missing debug redistributables.
+
+For the best experience build in Release mode.
```

Comment:
I think you should refer to info about PDB files and winbind below too, to inform about how to potentially make debug mode things work. (But the gist of it; stick to release mode if possible, is good to get across.)